client/dashboard/data/index.ts

```diff
@@ -304,13 +304,14 @@ export const fetchSiteSettings = async ( siteIdOrSlug: string ): Promise< SiteSe
 };
 
 export const updateSiteSettings = async ( siteIdOrSlug: string, data: Partial< SiteSettings > ) => {
-	return await wpcom.req.post(
+	const { updated } = await wpcom.req.post(
 		{
 			path: `/sites/${ siteIdOrSlug }/settings`,
 			apiVersion: '1.4',
 		},
 		data
 	);
+	return updated;
```

Comment:
Is this a personal preference change or something that was needed for some reason? Just curious.

Reply:
I updated it mainly to make it easier to type this 🙈 

```tsx
mutation: UseMutationResult< Partial< SiteSettings >, Error, Partial< SiteSettings >, unknown >;
```

https://github.com/Automattic/wp-calypso/pull/103426/files#diff-2ec7ed96eb26183de40eaf33e92e86819b50b575f1fec2b5074c8fa63d968d20R105

But in general, it's way easier to reason and work with if the query and the mutation return the same type/shape.